app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -120,7 +119,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
```

Comment:
That actually don't happen. If you tag someone then you are inside ArrowActivity. And to see the person you tagged you would navigate to ArrowTaggedActivity which would make this code to trigger for the first time and load all the tagged organizers including the new ones. 

If you want to reproduce the bug I'm talking about it's quite easy. 
Go to Arrow. Tap on the organizers list. Hit home button. Come back to the app.